src/com/amplitude/api/DatabaseHelper.java

```diff
@@ -21,6 +22,72 @@
 
 class DatabaseHelper extends SQLiteOpenHelper {
 
+    private static class DatabaseCorruptionHandler implements DatabaseErrorHandler {
```

Comment:
can we just extend DefaultDatabaseErrorHandler and call super in onCorruption?

Reply:
DefaultDatabaseErrorHandler is a final class, cannot extend :(

Reply:
Found a way to delegate to an instance of it 👍